### PR TITLE
[ch19] integrate frontend + backend multiproject gradle builds.

### PR DESCRIPTION
**Pull request by gungorkocak**
This PR is still lacks required additions to Readme.md, so I am going to add that.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "emeq-counter-frontend",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx parcel index.html",
+    "build": "npx parcel build index.html",
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (uralfrkn) [inline frontend/package.json]: The test step broke the build task. We need to write test script or remove test step from `build.gradle`.  
What do you think about that?

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "engines": {
+    "node": "12.13.0",
+    "npm": "6.12.0"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "elm-hot": "^1.1.4",
```
Comment (uralfrkn) [inline frontend/package.json]: Why we don't add `elm` to devDependencies?

File: .gitignore [hunk 3]
```diff
@@ -43,3 +43,10 @@ local.properties
 
 ### VS Code ###
 .vscode/
+
```
Comment (uralfrkn) [inline .gitignore]: In my opinion, we need to add the files to ignore files that created while running the Gradle tasks.

Comment (uralfrkn) [reply .gitignore]: That files;
```suggestion
backend/build.txt
backend/install.txt
backend/test.txt
build.txt
frontend/build.txt
frontend/install.txt
frontend/test.txt
install.txt
test.txt
```

File: frontend/package.json (hunk 2 above)
Comment (gungorkocak) [reply frontend/package.json:20]: Parcel handles elm build step with 'elm-hot' and 'node-elm-compiler' AFAIK. So we don't need to add elm itself to dependencies.

File: .gitignore (hunk 3 above)
Comment (gungorkocak) [reply .gitignore:46]: Done.

**Comment by gungorkocak** [comment]
@uralfrkn all comments are addressed, master branch is merged, and readme update added.
Please review again, and approve if you are up for it.

File: frontend/package.json (hunk 2 above)
Comment (uralfrkn) [reply frontend/package.json:20]: Thank you for this detail.

**Pull request merged**